src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html

```diff
@@ -169,3 +194,36 @@
     <ng-template ngPluralCase="other">{{ title }}s</ng-template>
   </span>
 </ng-template>
+
+<ng-template #dropdownTemplate>
+  <ng-container *ngFor="let data of dropdownData?.total.category | keyvalue">
+    <li class="list-group-item">
+      <div class="d-flex pb-2 pt-2">
+        <div class="ms-5 me-auto">
+          <span *ngIf="data.value.total"
+                [ngPlural]="data.value.total"
+                i18n>
+              {{ data.value.total }}
+            <ng-template ngPluralCase="=0">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="=1">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="other">{{ hwNames[data.key] | pluralize }}</ng-template>
+          </span>
+        </div>
+        <span [ngClass]="data.value.error ? 'me-2' : 'me-4'">
+          {{ data.value.ok }}
+          <i class="text-success"
+             *ngIf="data.value.ok"
+             [ngClass]="[icons.success]">
+          </i>
+        </span>
+        <span *ngIf="data.value.error"
+              class="me-4 ms-2">
```

Comment:
if you remove this `ms-2` and for the above `span`, add a class='me-2', does that work?

Reply:
If I understood your suggestion, then you end up with a shorter margin on the right, I don't think it's desired

![image](https://github.com/ceph/ceph/assets/106810838/aa622091-c9a7-4e81-8567-abbafb8b28ed)